tinyagentos/containers/lxc.py

```
@@ -65,14 +65,60 @@ def close(self) -> None:
+    """Return (project, state) for *container* by searching ALL incus projects.
+
+    ``incus exec`` targets the client's default project only, so a container
+    that lives in ``default`` while the client default is a per-user project
+    (e.g. ``user-999``) would be reported as "Instance not found". The async
+    path uses ``_resolve_container_project``; this is the synchronous sibling
+    the PTY path needs. Returns (None, None) if the container is not found or
+    the lookup fails, so the caller falls back to the default project.
+    """
+    try:
+        result = subprocess.run(
+            ["incus", "list", "--all-projects", "-f", "json"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)